doc/platform_api/CMIS_Diagnostic_Monitoring_Overview_in_SONiC.md

```diff
@@ -0,0 +1,2259 @@
+    lasertemplowwarning          = FLOAT                            ; laser temperature low warning threshold in Celsius
+```
+
+#### 2.1.3 Transceiver DOM flag data
+
+The `TRANSCEIVER_DOM_FLAG` table stores the flag status for the DOM data.
+
+lane_num: Represents lane number of the field. The lane number is an integer value that ranges from 1 to 8.
+
+```plaintext
+    ;Defines Transceiver DOM flags for a port
+    key                          = TRANSCEIVER_DOM_FLAG|ifname    ; information module DOM flags on port
+    ; field                      = value
+    temphighalarm_flag                    = BOOLEAN            ; temperature high alarm flag 
+    temphighwarning_flag                  = BOOLEAN            ; temperature high warning flag
+    templowalarm_flag                     = BOOLEAN            ; temperature low alarm flag
+    templowwarning_flag                   = BOOLEAN            ; temperature low warning flag
+    vcchighalarm_flag                     = BOOLEAN            ; vcc high alarm flag
+    vcchighwarning_flag                   = BOOLEAN            ; vcc high warning flag
+    vcclowalarm_flag                      = BOOLEAN            ; vcc low alarm flag
+    vcclowwarning_flag                    = BOOLEAN            ; vcc low warning flag
+    txpowerhighalarm_flag{lane_num}       = BOOLEAN            ; tx power high alarm flag
+    txpowerlowalarm_flag{lane_num}        = BOOLEAN            ; tx power low alarm flag
+    txpowerhighwarning_flag{lane_num}     = BOOLEAN            ; tx power high warning flag
+    txpowerlowwarning_flag{lane_num}      = BOOLEAN            ; tx power low alarm flag
+    rxpowerhighalarm_flag{lane_num}       = BOOLEAN            ; rx power high alarm flag
+    rxpowerlowalarm_flag{lane_num}        = BOOLEAN            ; rx power low alarm flag
+    rxpowerhighwarning_flag{lane_num}     = BOOLEAN            ; rx power high warning flag
+    rxpowerlowwarning_flag{lane_num}      = BOOLEAN            ; rx power low warning flag
+    txbiashighalarm_flag{lane_num}        = BOOLEAN            ; tx bias high alarm flag
+    txbiaslowalarm_flag{lane_num}         = BOOLEAN            ; tx bias low alarm flag
+    txbiashighwarning_flag{lane_num}      = BOOLEAN            ; tx bias high warning flag
+    txbiaslowwarning_flag{lane_num}       = BOOLEAN            ; tx bias low warning flag
+    lasertemphighalarm_flag               = BOOLEAN            ; laser temperature high alarm flag
+    lasertemplowalarm_flag                = BOOLEAN            ; laser temperature low alarm flag
+    lasertemphighwarning_flag             = BOOLEAN            ; laser temperature high warning flag
+    lasertemplowwarning_flag              = BOOLEAN            ; laser temperature low warning flag
+```
+
+#### 2.1.4 Transceiver DOM flag change count data
+
+The `TRANSCEIVER_DOM_FLAG_CHANGE_COUNT` table stores the flag change count for the DOM flags.
```

Comment:
@mihirpat1 Nit, do we need to add `chg_cnt` in every field name? leave up to you...

Reply:
@prgeor I have removed the `chg_cnt` in from all the field names.